src/content/en/2021/cms.md

```diff
@@ -2,19 +2,615 @@
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: CMS
 description: TODO
```

Comment:
@alonkochba could you add a description?

Reply:
We also have a fairly standard format of these (start with “The 2021 CMS chapter of the Web Almanac covering…”). Have a look at previous years for examples.

Reply:
It actually seems like the description always starts with "CMS chapter of the 2021 Web Almanac", for example the [2021 performance](https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/src/content/en/2021/performance.md) or [2020 CMS](https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/src/content/en/2020/cms.md) chapters.

I updated the description accordingly, but feel free to rephrase

Reply:
No that's fine - was doing it from memory on the phone :-)